raster-foundry/raster-vision-plugin
# Add RfLayerAnnotationGroupLabelSource

Overview
-----

This PR adds a label source using annotation groups on project layers.

Testing
-----

- from the repo root, `docker/build` if you haven't done that before
- `AWS_PROFILE=raster-foundry docker/run --jupyter --aws`
- fire up the label source notebook
- fill in an RF staging refresh token
- run the cells
- no errors should happen and you should have an `ObjectDetectionLabels` object at the end

Closes #3
